src/epochs.lua

```diff
@@ -712,7 +727,7 @@ function epochs.distributeRewardsForEpoch(currentTimestamp)
 				)
 				if actualOperatorReward > 0 then
 					-- distribute the rewards to the gateway
-					balances.transfer(gatewayAddress, ao.id, actualOperatorReward)
+					balances.transfer(gatewayAddress, ao.id, actualOperatorReward, false) -- unsafe address should not be allowed to delegate to gateways
```

Comment:
This is true today, but might not be in the future. Is it important to assert that here?

Reply:
we get the linting error as the function expects this to be set

Reply:
i suppose you're correct. my thought process was that delegates are always"safe" addresses. but by introducing "allow-unsafe-addresses" anyone (including unsafe addresses) can potentially delegate, meaning rewards should be allowed to be delegated to those addresses. if this is false, it will prevent the distribution to those unsafe addresses, which is not what we want. will update.

Reply:
[fd61aac](https://github.com/ar-io/ar-io-network-process/pull/328/commits/fd61aacf458304d83657bf3bd80593d4ce8b1f1d)